### DISCUSSION TITLE
Should we fork the Official Themes of HyDE?

### DISCUSSION THREAD

**Discussion by kRHYME7** [prasanthrangan/hyprdots#1971] [Q&A]
As a collaborator to this repo I can partially manage hyprdots. It's been fine for a while now. 

Now we are having breaking changes in the latest hyprland. We can fix issues regarding this if we can update the themes from @prasanthrangan's repo. But no one had access to that repo.

That's why I want to have your insights if it's fine to adapt the official themes and link it to the Hyde Project org while Tittu is AFK.

Basically we will just fork his theme repo and edit the links to the fork. Cool?

#1967

**Comment by rishav12s** [comment]
@kRHYME7 i think its a good idea as we would be able to update and fix these themes

some of themes have outdated gtk themes that have gotten updates which fixes certain aspects 

as well kitty theme syntax highlighting in synthwave etc

**Reply by kRHYME7** [reply]
Furthermore to the fixes, If we won't fork those themes..New users that will be using v0.45.0 will have  the impression that hyprdots is borke.

**Reply by rishav12s** [reply]
yes the shadow format changing will throw errors if not handled and updated for those themes

forking them will let contributors to make necessary/needed changes as well

**Reply by SketchyStunts** [reply]
Pretty much have every theme installed, official & non. Installed hyprland-git & made changes to all the themes new shadow variable manually to get it out the way. Which ones have outdated GTK themes & broken kitty syntax highlighting?

**Reply by kRHYME7** [reply]
@SketchyStunts  to fix those you can import the themes from here 
` https://github.com/HyDE-Project/hyde-themes ` 

0.45.0 is on testing rightnow not yet in extra therefore it's not merge yet. 

![image](https://github.com/user-attachments/assets/262320e0-56d9-4e90-8532-e0df9ae536b5)

![image](https://github.com/user-attachments/assets/0a2ed7f6-5da0-448a-9c27-5ebd3eefa29d)

**Comment by whoisabdi** [comment]
Maybe you could fork this repo as well to make some changes to the dotfiles and change dependencies since you can't do it in this repo because it's Prasanth's dotfiles after all.

You could maybe merge some awesome pull requests from this repo that can't be merged without adding more dependencies. I specially like the suggested changes about ~/.zshrc and adding new plugin manager like Sheldon to make the terminal load faster.

**Comment by kRHYME7** [comment]
Remind me to remove all `layout  =  ` specifics in the themes. these should not exist here at all
